This is a hack to address an underlying problem with vte on 11.3,
whereby the following write to stderr causes the window to close after
the subprocess has exited with code 127.  This doesn't happen if gmake
is launched from xterm nor in vte on 11.4.

Note that the subprocess's write to stderr, if any, will have already
completed successfully at this point, so we will still get that.

--- make-4.3/src/function.c.~1~	2020-01-03 07:11:27.000000000 +0000
+++ make-4.3/src/function.c	2020-11-24 19:31:15.303738019 +0000
@@ -1861,6 +1861,7 @@
       }
     shell_function_pid = 0;
 
+#ifdef EXIT_CODE_127_NOTIFICATION
     /* shell_completed() will set shell_function_completed to 1 when the
        child dies normally, or to -1 if it dies with status 127, which is
        most likely an exec fail.  */
@@ -1873,6 +1874,7 @@
         fflush (stderr);
       }
     else
+#endif
       {
         /* The child finished normally.  Replace all newlines in its output
            with spaces, and put that in the variable output buffer.  */
